src/utils/aepp-wallet-communication/rpc/wallet-rpc.js

```diff
@@ -69,6 +69,33 @@ const REQUESTS = {
       }
     )
   },
+  [METHODS.bridge] (callInstance, instance, client, { name }) {
+    // Authorization check
+    if (!client.isConnected()) return { error: ERRORS.notAuthorize() }
+
+    // wait for approval
+    client.updateInfo()
```

Comment:
why it is called? I can't get the comment

Reply:
Removed it.